service/src/test/java/bio/terra/profile/pact/provider/BPMProviderTest.java

```diff
@@ -83,6 +90,16 @@ void verifyPact(PactVerificationContext context) {
     context.verifyInteraction();
   }
 
+  @State("BPM is ok")
+  void getBpmStatus() {
+    when(samService.status()).thenReturn(new SystemStatusSystems().ok(true));
+    // For CloudSQL return value
+    doReturn(jdbcTemplate).when(namedParameterJdbcTemplate).getJdbcTemplate();
+    doReturn(true).when(jdbcTemplate).execute(any(ConnectionCallback.class));
+    // Checks status of services and stores result for the next time status endpoint is called
+    profileStatusService.checkStatus();
+  }
+
```

Comment:
FYI, this is what the pact file looks like that is generated by the Rawls consumer test:

```
{
  "consumer": {
    "name": "rawls-consumer"
  },
  "interactions": [
    {
      "description": "Request to BPM ok status",
      "providerStates": [
        {
          "name": "BPM is ok",
          "params": {
            "CloudSQL": "ok",
            "Sam": "ok"
          }
        }
      ],
      "request": {
        "headers": {
          "Accept": "application/json"
        },
        "method": "GET",
        "path": "/status"
      },
      "response": {
        "body": {
          "ok": true,
          "systems": {
            "CloudSQL": {
              "ok": true
            },
            "Sam": {
              "ok": true
            }
          }
        },
        "headers": {
          "Content-type": "application/json"
        },
        "matchingRules": {
          "body": {
            "$.ok": {
              "combine": "AND",
              "matchers": [
                {
                  "match": "type"
                }
              ]
            },
            "$.systems.CloudSQL.ok": {
              "combine": "AND",
              "matchers": [
                {
                  "match": "type"
                }
              ]
            },
            "$.systems.Sam.ok": {
              "combine": "AND",
              "matchers": [
                {
                  "match": "type"
                }
              ]
            }
          }
        },
        "status": 200
      }
    }
  ],
  "metadata": {
    "pact-jvm": {
      "version": "4.4.0"
    },
    "pactSpecification": {
      "version": "3.0.0"
    }
  },
  "provider": {
    "name": "bpm-provider"
  }
}